util/src/main/java/io/camunda/zeebe/util/sched/ActorTask.java

```diff
@@ -63,7 +59,6 @@ public ActorTask(final Actor actor) {
   /** called when the task is initially scheduled. */
   public ActorFuture<Void> onTaskScheduled(
       final ActorExecutor actorExecutor, final ActorThreadGroup actorThreadGroup) {
-    this.actorExecutor = actorExecutor;
```

Comment:
:wrench: Can we remove the related parameter as well?

Reply:
Whoops, nice catch!